crates/db_schema/src/impls/post.rs

```diff
@@ -149,6 +149,7 @@ impl Post {
     pool: &mut DbPool<'_>,
     for_creator_id: PersonId,
     for_community_id: Option<CommunityId>,
+    for_instance_id: Option<InstanceId>,
     removed: bool,
   ) -> Result<Vec<Self>, Error> {
```

Comment:
This works nicely to remove posts in communities from a given instance if the user was banned from there. Unfortunately the same thing doesnt work for comments because there is no column `comment.instance_id`, and diesel doesnt support joins in update queries. So to handle removals for comments in this case, it is either necessary to add that column, or run two queries (first select the comments and then update them). What do you think?

Reply:
Just a TODO here for now because I already created a helper function for comment removals in my pagination branch.